.github/workflows/test_kilosort4.yml

```diff
@@ -0,0 +1,74 @@
+name: Testing Kilosort4
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 12 * * 0"  # Weekly on Sunday at noon UTC
+  pull_request:
+    types: [synchronize, opened, reopened]
+    branches:
+      - main
```

Comment:
@JoeZiminski I would remove this, otherwise it will run on any open PR...no?

Reply:
@alejoe91 oh yes good point, should this be a weekly cron job like [here](https://github.com/SpikeInterface/spikeinterface/blob/694f8620e7e79cc914678bf2c8a3a7d70df3af29/.github/workflows/all-tests.yml#L6)?

so just

```
on:
  workflow_dispatch:
  schedule:
    - cron: "0 12 * * 0"  # Weekly on Sunday at noon UTC
```

Reply:
Actually, there is a conflict with exposing `save_preprocesed_copy`, will fix now